orchestration/cleanup_completed_agents.py

```
@@ -0,0 +1,204 @@
+"""
+TMux Agent Cleanup Utility
+
+This script identifies and cleans up completed tmux agents that are sitting idle.
+Agents are considered completed if they have completion markers in their logs.
+"""
+
+import os
+import subprocess
+import json
+import time
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)